helm/provisioner/templates/daemonset_linux.yaml

```diff
@@ -28,6 +28,9 @@ spec:
 {{ .Values.podAnnotations | toYaml | trim | indent 8 }}
 {{- end }}
     spec:
+{{- if .Values.hostPID }}
+      hostPID: true
```

Comment:
Nit: Maybe this should be:

```
hostPID: {{ .Values.hostPID }}
```

The default value in the reference is `false` https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#podspec-v1-core, also the value set from the chart is `false` so I think the above would work.